src/app/pages/offline-map/metadata.model.ts

```diff
@@ -2,6 +2,13 @@ export interface PackageMetadata {
     name: string;
     lastModified: string;
     xyz: number[];
-    url: string;
-    sizeInMb: number;
+    urls: string[];
+    sizeInMib: number;
+  }
+
+  export interface PackageMetadataCombined {
```

Comment:
Er dette for en pakke som er delt i flere deler? Litt vanskelig å skjønne utfra navnet. Hva med navnet MultiPartPackageMetadata, CompoundPackageMetadata eller noe i den dur?